api/gateway.proto

```diff
@@ -136,7 +144,10 @@ message Gateway {
   // This is either an auth token (such as an API Key) or a TLS private certificate.
   // Requires the RIGHT_GATEWAY_READ_SECRETS for reading and RIGHT_GATEWAY_WRITE_SECRETS for updating this value.
   Secret lbs_lns_secret = 22 [(gogoproto.customname) = "LBSLNSSecret"];
-  // next: 23
+  // The authentication code for gateway claiming.
+  // Requires the RIGHT_GATEWAY_READ_SECRETS for reading and RIGHT_GATEWAY_WRITE_SECRETS for updating this value.
```

Comment:
Looking at the implementation, you can't select/update sub-fields only (reading/writing the timestamps without reading/writing the value), so we need to make that clear in the documentation. Also, we should then update the field mask validation to exclude those sub paths in `pkg/ttnpb/field_mask_validation.go`.

Reply:
Ok I guess you mean [this](https://github.com/TheThingsNetwork/lorawan-stack/blob/7543bf6d423096650564d188be357d937f9b7384/pkg/ttnpb/field_mask_validation.go#L864-L887)?